packages/oid4vci-demo-frontend/src/utils/ObjectUtils.ts

```diff
@@ -0,0 +1,16 @@
+import {DataFormRow} from "../ecosystem-config";
+
+export const extractKeys = (data: DataFormRow[]): string[] => {
+    return data.flat().map(field => field.key);
+};
+
+export const extractRequiredKeys = (data: DataFormRow[]): string[] => {
+  return data.flat().filter(field=> field.required).map(field => field.key);
+};
+
+export const transformDataToObject = (data: DataFormRow[]): Record<string, string> => {
+    return data.flat().reduce((acc, field) => {
+        acc[field.key] = '';
```

Comment:
Why? Also I would expect this to use a flatMap.

Reply:
replaced with flatmap. and for your question, I think you’re talking about assigning the empty string to the keys, we have been doing that for other form inputs such as firstName, lastName, and email (https://github.com/Sphereon-Opensource/OID4VC-demo/blame/16bcb3df57fdf1b804b25cc9f2dd396fbd9e5863/packages/oid4vci-demo-frontend/src/pages/SSIInformationRequestPage/index.tsx#L42).

Reply:
Okay then suggest to rename this method, because that name doesn't suggest it

Reply:
renamed it to `transformFormConfigToEmptyObject`